beacon-chain/sync/validate_beacon_blocks.go

```diff
@@ -92,6 +93,7 @@ func (s *Service) validateBeaconBlockPubSub(ctx context.Context, pid peer.ID, ms
 	s.pendingQueueLock.RUnlock()
 
 	if err := helpers.VerifySlotTime(uint64(s.chain.GenesisTime().Unix()), blk.Block.Slot, params.BeaconNetworkConfig().MaximumGossipClockDisparity); err != nil {
+		log.WithError(err).WithField("blockSlot", blk.Block.Slot).Debug("Rejecting incoming block.")
```

Comment:
Can we revert this line?

1.) It's odd to have `Rejecting incoming block` just for this time check. If we do want this then we should add this rejecting incoming back to every validation step. (Even for debug)

2.) This hasn't been relevant since we had the time bug

3.) Maybe we also can downgrade it to trace? and something like this for all the validation steps?

Reply:
Its pretty important to know if a block has been rejected, unlike attestations blocks occur only once every 12 seconds. Node operators will have very little idea if their local clocks are in sync with the rest of the network if these logs disappear. They would see poor attestation performance, voting on the wrong head etc but wouldn't be able to tie it in with an out of sync clock if this does happen. Rejecting a block happens rarely enough that it is fine to keep it in debug. On 2) , that isn't really true, a lot of users still see that log locally when their clock is out of sync.

Reply:
That's fair. My point is we should value consistency here. Not just have debug logging for 1-2 of the steps (because previous incidents) and not all the steps. What do you think about?

1.) Add debug logging for rest of the steps (e.g. block descendants, signature... etc)

2.) Conform to the same prefix. (e.g. `Could not validate incoming back: `)

Let me know if you want to do that here? I'm happy to do it as well either push it to this PR or open a new PR

Reply:
Sounds good :+1: , agree with standardizing logs for all our debug messages here otherwise its ugly looking. I can fix this tomorrow too, or you are welcome to also clean it up if you are free.

Reply:
Great. I can help. Thank you!